commonui/src/test/java/com/hyperwallet/android/ui/common/util/TransferMethodUtilsTest.java

```diff
@@ -151,29 +152,31 @@ public void getTransferMethodDetail_returnsPayPalDetails() {
                 "sunshine.carreiro@hyperwallet.com").build();
 
         String actual = getTransferMethodDetail(mContext, transferMethod, PAYPAL_ACCOUNT);
-        assertThat(actual, is("sunshine.carreiro@hyperwallet.com"));
+        MatcherAssert.assertThat(actual, Matchers.is("sunshine.carreiro@hyperwallet.com"));
```

Comment:
please import the whole static class for `MatcherAssert` and `Matchers` please

Reply:
removed